vignettes/options.Rmd

```diff
@@ -0,0 +1,37 @@
+---
+title: "logrx Options"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{logrx Options}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  echo = FALSE
+)
+```
+
+```{r setup}
+library(knitr)
+library(logrx)
+library(tibble)
+```
+
+One of the ways that `logrx` has been built to be customizable is to add a set of options to enable different logging and execution elements.
+
```

Comment:
Why would someone want this?  Maybe we should have a few examples/scenarios of when a user might want this type of functionally.  This shows my lack of understanding here ! :)

Reply:
I think this is a first draft and once we have the options all there we can take a second pass at this and either link to a vignette that explains and uses the option or write sections here

Reply:
Approved has a vignette that shows how to use it but not why to use it. I can add a why section to that vignette or we can put all option uses here.  I'm up for either and agree this would help.

For lint, I think it would be the same reason we have linting in our package development pipelines. A lot of teams might not be using git to manage the scripts they are executing, so they cannot leverage the nice CI/CD workflows that come with github and bitbucket.  This gives them the option for static code analysis that can be configured at an organization level to enforce best practice.